tests/cases/conformance/async/es2017/asyncAliasReturnType_es2017.ts

```diff
@@ -0,0 +1,6 @@
+// @target: es2017

```

Comment:
This test is unnecessary as its only purpose is to verify return type check for async functions, which is already handled by the existing tests. The reason we differ between es5/es6 is that the rules governing whether this is allowed are different when targeting es5/3, but will be the same for es6+